boltdesignsystem/bolt
# Hotfix/DS-330 Remove deprecated card

## Jira

https://pegadigitalit.atlassian.net/browse/DS-330

## Summary

Remove deprecated Card component.

## Details

Remove deprecated Card component, its docs, and a few mentions of this component in tests and build files. I renamed the docs folder for "Card (Replacement)" to "Card" but stopped there. I did not rename all the docs pages from "card-replacement-x" to "card-x", and I did not rename the `<bolt-card-replacement>` element itself. I figure that will happen later.

## How to test

Build is passing and there are no regressions to the "replacement" card. Spot check other components for any obvious regressions.

## Release notes

- **Bolt Card**: `@bolt-components-card/card.twig` has been removed in `v3.x`. Use `@bolt-components-card-replacement/card-replacement.twig` instead.
- **Bolt Band**: `items` prop has been removed in `v3.x`. If you are still using the `items` prop to add pinned content to a band, [see docs](https://v2-31-2.boltdesignsystem.com/pattern-lab/?p=components-band-with-pinned-content) on how to use the `pinned_content` prop instead.
